src/emulator/gui/src/settings_dialog.cpp

```diff
@@ -30,7 +30,43 @@ void draw_settings_dialog(HostState &host) {
     ImGuiTabBarFlags settings_tab_flags = ImGuiTabBarFlags_None;
     ImGui::BeginTabBar("SettingsTabBar", settings_tab_flags);
 
+    // Core
+    if (ImGui::BeginTabItem("Core")) {
+        ImGui::PopStyleColor();
+        ImGui::Text("LLE Module \nSet your LLE Module Wanted.");
+        if (ImGui::IsItemHovered())
+            ImGui::SetTooltip("Write exemple 'libpgf', press Apply and re write other.");
+        std::string lle_module;
+        ImGui::InputTextWithHint("Set lle module", "Add lle module wanted", &lle_module);
+        if (ImGui::Button("Apply add LLE Modules")) {
```

Comment:
if (ImGui::Button("Apply LLE Modules.")) {

Reply:
Is wrong, is not apply lle module is add news lle module on list ^^

Reply:
Little wrong, here is for Apply add lle inside list enable, so here said this is wrong